.github/scripts/check_event_type.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+java_events = set()
+with open("./metadata-io/src/main/java/com/linkedin/metadata/datahubusage/DataHubUsageEventType.java") as java_file:
+    for line in java_file:
+        if '''Event"''' not in line:
+            continue
+        line = line.replace("\n", "").split('"')[1]
+        java_events.add(line)
+
+ts_events = set()
+with open("././datahub-web-react/src/app/analytics/event.ts") as ts_file:
+    for line in ts_file:
+        if '''Event,''' not in line:
+            continue
+        line = line.replace(",\n", "").replace(" ", "")
+        ts_events.add(line)
+
+ts_events_not_in_java = ts_events.difference(java_events)
+if len(ts_events_not_in_java) > 0:
+    print(f"Missing {ts_events_not_in_java} from DataHubUsageEventType.java. Please add")
```

Comment:
Thanks for adding this. We will need to watch these usage indexes as they continue to grow with all of these new events!

Overall, I like the idea of keeping these in sync.